libcontainer/process_linux.go

```diff
@@ -271,6 +294,27 @@ func (p *initProcess) createNetworkInterfaces() error {
 	return nil
 }
 
+func runCmd(command configs.Command, statePath string) error {
+	cmd := exec.Command(command.Path, command.Args[:]...)
```

Comment:
@mrunalp is there a reason to use `exec.Command` instead of populating `exec.Cmd` directly with the args ?
`exec.Command` forces the argv[0] to be the path, and hence `command.Args` can only occupy argv[1]...

IMO, the following is better 
```
-       cmd := exec.Command(command.Path, command.Args[:]...)
+       cmd := exec.Cmd{
+               Path: command.Path,
+               Args: command.Args[:],
+       }
```